tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/ListDependencyTestCase.java

```diff
@@ -60,7 +60,7 @@ public void testListDependenciesOfModule() throws Exception {
                 "├── ballerina/io\n" +
                 "└── ballerina/math\n";
 
-        String actualOut = balClient.runMainAndReadStdOut("list", clientArgs, tempProjectDirectory.toString());
+        String actualOut = balClient.runMainAndReadStdOut("list", clientArgs, null, tempProjectDirectory.toString());
```

Comment:
I think it would be better to pass an empty map here and then we won't need the null check in the `runMainAndReadStdOut` method. Or we can create an overloaded method which will pass an empty map to the other overloaded method. In that case, we won't have to create an empty map as well.

Eg -
```java
public String runMainAndReadStdOut(String command, String[] args, Map<String, String> envProperties,
                                       String commandDir) throws BallerinaTestException {
    ...
}

public String runMainAndReadStdOut(String command, String[] args, String commandDir) throws BallerinaTestException {
    runMainAndReadStdOut(command, args, new HashMap<>(), commandDir);
}
```
WDYT?

Reply:
+1 to add another overloaded method.

Reply:
Added with 5ed76dd